lib/run.go

```diff
@@ -266,18 +266,29 @@ func genDeplist(acipath string, reg registry.Registry) ([]string, error) {
 }
 
 func (a *ACBuild) mirrorLocalZoneInfo() error {
-	zif, err := os.Readlink("/etc/localtime")
-	if err != nil {
+	var localtimefile = "/etc/localtime"
+
+	finfo, err := os.Lstat(localtimefile)
+	switch {
+	case os.IsNotExist(err):
+		return nil
+	case err != nil:
 		return err
+	case finfo.Mode()|os.ModeSymlink > 0:
```

Comment:
I'll suggest that you do an & operation. I think this condition would also match when finfo.Mode() is something else than ModeSymlink.

```
case finfo.Mode() & os.ModeSymlink == os.ModeSymlink:
```